rmgpy/solver/base.pyx

```diff
@@ -200,7 +200,21 @@ cdef class ReactionSystem(DASx):
         if surfaceReactions is None:
             surfaceReactions = []
             
-
+        if conditions:
+            isConc = hasattr(self,'initialConcentrations')
+            keys = conditions.keys()
+            if 'T' in keys and hasattr(self,'T'):
+                self.T = Quantity(conditions['T'],'K')
+            if 'P' in keys and hasattr(self,'P'):
+                self.P = Quantity(conditions['P'],'Pa')
+            for k in keys:
+                if isConc:
+                    if k in self.initialConcentrations.keys():
+                        self.initialConcentrations[k] = Quantity(conditions[k],'mol/m^3')
```

Comment:
I'm not too familiar with LiqReactor, but couldn't the conditions be specified in different units? If so, we're missing a conversion